<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <node pkg="mir_experiments" type="octomap_map_changes" name="octomap_tracking_server" args="$(find mir_experiments)/maps/tidy_small_depth_camera.bt" output="screen">
		<param name="resolution" value="0.05" />
		
        <param name="frame_id" type="string" value="odom" />
        <param name="base_frame_id" type="string" value="odom" />
        <param name="change_id_frame" type="string" value="odom" />
        <param name="topic_changes" value="/octomap_tracking_server/changeset" />
        <param name="filter_ground" type="bool" value="true" />

        <param name="min_change_pub" value="0" />
		<!-- data source to integrate (PointCloud2) -->
        <remap from="cloud_in" to="/camera/depth/points" />
        <remap from="free_cells_vis_array" to="/octomap_tracking_server/free_cells_vis_array" />
        <remap from="occupied_cells_vis_array" to="/octomap_tracking_server/occupied_cells_vis_array" />
        <remap from="octomap_binary" to="/octomap_tracking_server/octomap_binary" />
        <remap from="octomap_full" to="/octomap_tracking_server/octomap_full" />
        <remap from="octomap_point_cloud_centers" to="/octomap_tracking_server/octomap_point_cloud_centers" />
        <remap from="projected_map" to="/octomap_tracking_server/projected_map" />
	
    </node>


</launch>

